### ISSUE TITLE
Anilist's tracker can't find new titles

### ISSUE THREAD

**Issue by AriaDeAzure** [mihonapp/mihon#926]
### Steps to reproduce

When you try to add a new manga/manhwa to the tracker, it can't find the title. It still works with older titles (added to Anilist before April, if my calculation is correct), but I haven't been able to add anything from the newer titles (added from April till now).

![image](https://github.com/mihonapp/mihon/assets/104279446/74f3edaa-bb59-4f80-9b50-cd35d8aeb028)
For example, this manhwa is on Anilist ([proof](https://anilist.co/manga/176959/Guwon-Bangjeongsik/)) and it was added maybe around two months ago, but I can't find it using the tracker. The title I'm searching for is the English title and it's a match. I also tried the Korean title but no luck.

![image](https://github.com/mihonapp/mihon/assets/104279446/f1f6b880-5ea8-4f52-a62b-9e2c85e6bef4)
Another example is this manhwa that was added to Anilist in the past couple of days ([proof](https://anilist.co/manga/178045/Hongcha-Jeongwonui-Gwihan-Agassiimnida/)). I can't find it with the English title or the Korean title.

### Expected behavior

When adding a new title to the tracker, it should return a list with all the titles that match the one you are looking for (including the English/Japanese/Korean/Chinese titles).

### Actual behavior

If it's a newer title on Anilist, it doesn't return a thing and it says "No result was found".

### Mihon version

0.16.5

### Android version

Android 11

### Device

Samsung Galaxy Tab A

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by BrutuZ** [comment]
https://anilist.co/forum/thread/72975
tldr; Not our problem

**Comment by AriaDeAzure** [comment]
> https://anilist.co/forum/thread/72975 tldr; Not our problem

I'm not asking to add new titles to Anilist. I know they have nothing to do with Mihon. These titles are on Anilist (I even added proof), but I can't find them using Mihon's tracker.

**Comment by BrutuZ** [comment]
Good job reading the title, now go for the message too

**Comment by AriaDeAzure** [comment]
Nothing on those messages says that you can't search for new titles. It says they are closing the submissions to fix some issues they have (which they do from time to time). Nevertheless, the older titles seem to still be working with the tracker and they have been adding some titles now and then. Given that, my assumption it could be a bug or some change with the API was understandable. The snarkiness was uncalled for. Thank you, either way.

**Comment by BrutuZ** [comment]
It does though, says submissions are closed *because the search is broken*.

I linked that because they only have an official/pinned thread regarding the effect (submissions closed) but not one for the cause (search issue), that's scattered over issue reports by users (the numbered links) and individual replies
